src/language-html/utils/index.js

```diff
@@ -364,7 +364,10 @@ function _inferScriptParser(node) {
     return "html";
   }
 
-  if (type && (type.endsWith("json") || type.endsWith("importmap"))) {
+  if (
+    (type && (type.endsWith("json") || type.endsWith("importmap"))) ||
```

Comment:
Not related, but is `importmap` check a mistake? I think it should be  `=== "importmap"`

Reply:
Maybe this the `json` mime check should be endsWith `/json` or `+json`?

Reply:
> Not related, but is importmap check a mistake? I think it should be === "importmap"

Maybe for `systemjs-importmap`. https://github.com/systemjs/systemjs/blob/main/docs/import-maps.md#loading-import-maps